src/compiler/crystal/macros/methods.cr

```diff
@@ -400,6 +400,24 @@ module Crystal
         interpret_check_args { symbolize }
       when "class_name"
         interpret_check_args { class_name }
+      when "docs"
```

Comment:
I'm wondering if this should better be called `doc` in accordance with `ASTNode#doc`.

Reply:
I thought about it, but ultimately settled on `docs` because I thought it reads better 🤷. Don't really have strong feelings either way tho.